base/linalg/triangular.jl

```diff
@@ -376,6 +376,8 @@ scale!(c::Number, A::Union{UpperTriangular,LowerTriangular}) = scale!(A,c)
 ######################
 
 A_mul_B!(A::Tridiagonal, B::AbstractTriangular) = A*full!(B)
+A_mul_B!(C::AbstractMatrix, A::AbstractTriangular, B::Tridiagonal) = A_mul_B!(C, full(A), B)
```

Comment:
I am not sure either. However I need to define these A_mul_B! to resolve definition ambiguity. Do you have a suggestion what to use instead of full? Full is also not that bad here since I expect AbstractTriangular to be half full anyway.